test/requests/document_link_expectations_test.rb

```diff
@@ -20,20 +20,37 @@ def map_expectations(expectations)
     end
   end
 
+  class FakeHTTPResponse
+    attr_reader :code, :body
+
+    def initialize(code, body)
+      @code = code
+      @body = body
+    end
+  end
+
   def run_expectations(source)
     document = RubyLsp::Document.new(source)
-    RubyLsp::Requests::DocumentLink.new("file://fake/path/without_version.rb", document).run
+    js_content = File.read(File.join(TEST_FIXTURES_DIR, "rails_search_index.js"))
+    fake_response = FakeHTTPResponse.new("200", js_content)
+    T.unsafe(Net::HTTP).stub(:get_response, fake_response) do
```

Comment:
I think if you add `require "minitest/mock"` to `sorbet/tapioca/require.rb` and re-generate the RBI for minitest, you won't need the unsafe.